mangaki/mangaki/views.py

```diff
@@ -141,6 +141,41 @@
     'endpoint': reverse_lazy('vote')
 }
 
+def deep_dict_merge(source: Dict, destination: Dict,
```

Comment:
I really feel like this method was overkill but that's okay.
You could just have set `ctx['meta'][<key>] = <value>`.